worlds/pokemon_crystal/__init__.py

```diff
@@ -310,7 +312,7 @@ def get_random_trap():
             if item_code > 0:
                 self.itempool.append(self.create_item_by_code(item_code))
             else:  # item is NO_ITEM, trainersanity checks
-                self.itempool.append(self.create_item_by_const_name(get_random_filler_item(self.random)))
+                self.itempool.append(self.create_item_by_const_name(get_random_filler_item(self.random, self)))
```

Comment:
```suggestion
                self.itempool.append(self.create_item_by_const_name(get_random_filler_item(self)))
```
Can just pass the `World` here